odata-csdl/15 Identifier and Path Values.md

```diff
@@ -544,12 +544,15 @@ types
 11. SHOULD NOT include new/unknown values for the
 [`AppliesTo`](#Applicability) attribute
 :
-12. MAY include new CSDL annotations
+12. SHOULD NOT include constant [Geo](#GeoValues) or [Stream values](#StreamValues) in annotations
+13. MAY include new CSDL annotations
 
 In addition, OData 4.01 services:
```

Comment:
```suggestion
In addition, OData 4.01 or 4.02 services:
```
Are there other places where a mention of 4.01 must be changed to include 4.02 as well?

Reply:
4.02 services are the ones that MAY include Geo & Stream constants, only the lower versions SHOULD NOT.

Second: this is the first 4.02 issue, so no other places yet.

Reply:
Before and after this PR, we say that
> OData 4.01 services SHOULD NOT have identifiers within a uniqueness scope.

Does that not apply to 4.02 services?